habitat-aspnet-sample/hooks/init

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+#
+ln -sf {{pkg.path}}/www {{pkg.svc_var_path}}
```

Comment:
Pretty much everything in these hooks (init and run) that's in a `{{}}` should be double-quoted to avoid word splitting.